src/sandbox/alternative-threading/CountDownLatch2.cs

```diff
@@ -77,7 +77,7 @@ public bool await()
 		/// Waits the specified amount of time for the latch
 		/// returning true if the latch was acquired
 		/// <summary>
-        public bool await(TimeSpan timeout)
+        public bool Await(TimeSpan timeout)
```

Comment:
This was causing some issues because of the reserved keyword `await` (in the AsyncAwait TPL)